### PR TITLE
GH-72 Cooked v3

### PR DESCRIPTION
**Pull request by vLuckyyy**

### CODE REVIEW

**Comment by CitralFlo** [comment]
The only issue I see:
image
It disturbs the whole content layout

**Review by CitralFlo — APPROVED** [review]
The code looks good, Site is responsible and looks CLEAN 🔥

File: app/api/project/route.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { NextResponse } from "next/server";
+
+export async function GET(request: Request) {
+  const res = await fetch(
+    `https://cms.eternalcode.pl/api/projects?populate=*`,
```
Comment (Hyd3r1) [inline app/api/project/route.ts:5]: When u use .env, please use it in whole code

File: app/api/team/route.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { NextResponse } from "next/server";
+
+export async function GET(request: Request) {
+  const res = await fetch(
+    `https://cms.eternalcode.pl/api/team-members?populate=*`,
```
Comment (Hyd3r1) [inline app/api/team/route.ts:5]: Again, what if we change URL? 🤷‍♂️

**Review by Hyd3r1 — CHANGES_REQUESTED** [review]

**Pull request merged**